src/OpenTelemetry/CHANGELOG.md

```diff
@@ -29,9 +29,11 @@
   * `telemetry.sdk.name` = `opentelemetry`
   * `telemetry.sdk.language` = `dotnet`
   * `telemetry.sdk.version` = [SDK version]
-* `Resource` constructor marked as internal, as `ResourceBuilder` is
-  the recommended API to build resources.
+* `Resource` constructor marked as internal, as `ResourceBuilder` is the
+  recommended API to build resources.
   ([#1566](https://github.com/open-telemetry/opentelemetry-dotnet/pull/1566))
+* Added check for ActivityTraceFlags for export
```

Comment:
Lets also add "Changed BaseExportProcessor to have it override OnExport instead of OnEnd", as this is a key breaking change.

Reply:
Fixed this.